### DISCUSSION TITLE
Need Help for OpenRouter LLM list Update

### DISCUSSION THREAD

**Discussion by Zerglingzl** [langgenius/dify#11582] [Help]
### Self Checks

- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] [FOR CHINESE USERS] 请务必使用英文提交 Issue，否则会被关闭。谢谢！:）
- [X] Please do not modify this template :) and fill in all the required fields.

### 1. Is this request related to a challenge you're experiencing? Tell me about your story.

How could I keep my openRouter LLM list update to the latest? there are quite more new models, even the newest Gemini 2 Flash. I saw it on OpenRouter website and tried in playground mode, so it indeed exists. BUT, when I open LLM configuration of OpenRouter on Dify, there still old list...

what should I do? keep waiting, or do something? pls help, thanks a lot!

**Comment by xceleratortech** [comment]
Still no 2.0 flash in list
